bin/kano-capture-logs

```diff
@@ -25,8 +28,13 @@ def store_logging_error():
 
 
 def get_logs():
-    rc = os.system("USER=root LOGNAME=root HOME=/root kano-feedback-cli {} >{} 2>&1".format(
-        LOG_DEST, LOG_OUTPUT))
+    rc = os.system(
+        'USER=root LOGNAME=root HOME=/root'
+        ' kano-feedback-cli --output {} --screenshot'
```

Comment:
Given this happens at boot, I'm not sure it's worth doing a screenshot - on the other hand it's also pretty harmless.

Reply:
It used to be there, but the `--title` tag will be more useful to know where it's coming from. Leaving it there obviously makes it slower to bootup into Safe Mode, and the archive a bit bigger. I'll remove it 👍